UNL-CSE-AMBASSADORS/VITA
# Dependent/Language Information

Added in client language and dependents information to the private queue. 

**TESTING**
* Ensure the public queue still works
* Sign into Site admin account
* Go sign up for an appointment and add like 5 dependents
* Go to private queue, find the appointment you just created, ensure you see language and the dependents listed
  * Ensure you see the last name of the dependents
* Sign into preparer account
* Go to private queue, find the appointment you just created, ensure you see language and the dependents listed
  * Ensure you do not see the full last name of the dependents
* Ensure that appointments without dependents still show properly, and that the dependents heading is not there
